src/app/GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -145,7 +145,9 @@ private void Done()
 
                 if (!Visible)
                 {
-                    ShowDialog(OwnerWindow);
+#pragma warning disable WFO5002 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+                    await ShowDialogAsync(OwnerWindow);
```

Comment:
Why change this?

Reply:
Unless there's a compelling reason, I'd just suppress VSTHRD103 and leave the code as is

Reply:
There is no requirement to change this, I just found VSTHRD103  reasonable.